cloud-aws/src/main/resources/templates/aws-cf-stack.ftl

```diff
@@ -495,6 +495,27 @@
 
     <#if enableEfs>
     ,
+    "MountTargetSecurityGroup": {
+      "Type": "AWS::EC2::SecurityGroup",
+      "Properties": {
+        <#if existingVPC>
+          "VpcId" : { "Ref" : "VPCId" },
+        <#else>
+          "VpcId" : { "Ref" : "VPC" },
+        </#if>
+        "GroupDescription": "Security group for EFS mount target",
+        "SecurityGroupIngress": [
```

Comment:
this can easily hit customer issues when we havea limited policy

Reply:
The allowed IP address should cover all IP addresses in VPC, and right now, only EC2 from the Data Lake should access them. Do you see what kind of access could fail?

What is your suggestion?

Reply:
From Lajos "the "pre-existing SG" case is what is problematic here; we shall not attempt creating a new SG for the EFS mount target if there are existing SGs choosen by the customer."

I discussed with Lajos offline and realize Richard concerns about creating security group if user provides security group. Updated the CloudFormation template.